### ISSUE TITLE
Settle on a way to handle body size limit on proxy requests

### ISSUE THREAD

**Issue by deifyed** [oslokommune/gatekeeper#39]
When sending big requests (files/pictures/videos) through the proxy, Gatekeeper will block the request with an 413 request entity too large error. The limit is per now set to 1mb.  

I suggest (without thinking about it really) that we set the limit to infinity, or something really high and let reverse proxies handle request limits. A counter argument is that not everyone uses reverse proxies, but I suggest we implement it when and if its needed. Debugging request limiting is a pain in the. .

Any thoughts on how we should handle this?

**Comment by yngvark** [comment]
> let reverse proxies handle request limits

Not quite sure I understand this. The Gatekeeper is a reverse proxy for a destination/upstream/backend, so I'm assuming you mean let the upstream handle request limits - which makes sense to me, so we can set a limit one place instead of two. And it also makes sense to let the upstream decide this, instead of having some value in the Gatekeeper, which may or may not work for all the X upstreams the Gatekeeper serves.

> "not everyone uses reverse proxies" 

Not sure what this means?

Maybe a potential pitfall with unlimited limit is that it allows posting huge payloads to the gatekeeper, filling its memory. Though I don't think this is an issue, as Express uses streaming by default: https://github.com/villadora/express-http-proxy#streaming, as I guess any sane proxy should.

**Comment by deifyed** [comment]
What I mean by reverse proxies is either an k8s ingress or just an nginx/apache/* instance in front of the Gatekeeper.

Your described pitfall is indeed a problem, so a limit should exist - but I argue it should be in the ingress or any other reverse proxy

**Comment by yngvark** [comment]
Right, I see. I agree, either the reverse proxy or the upstream can set the limit.
